apps/electionguard-cli/Encrypt/EncryptCommand.cs

```diff
@@ -19,19 +19,24 @@ public static Task Encrypt(EncryptOptions encryptOptions)
         private async Task EncryptInternal(EncryptOptions encryptOptions)
         {
             encryptOptions.Validate();
-            if (string.IsNullOrEmpty(encryptOptions.Context)) throw new ArgumentNullException(nameof(encryptOptions.Context));
-            if (string.IsNullOrEmpty(encryptOptions.Manifest)) throw new ArgumentNullException(nameof(encryptOptions.Manifest));
-            if (string.IsNullOrEmpty(encryptOptions.BallotsDir)) throw new ArgumentNullException(nameof(encryptOptions.BallotsDir));
+            if (string.IsNullOrEmpty(encryptOptions.Context))
+                throw new ArgumentNullException(nameof(encryptOptions.Context));
```

Comment:
just making it easier to read on a narrow screen. the auto linting wasnt working for me on this file right now

Reply:
it's more readable on 2 lines anyway